docs/tutorials/integrations/kserve-mm/blue-green.md

```diff
@@ -128,153 +124,113 @@ To send inference requests to the model:
     | grep -e app-version
     ```
 
-??? note "Sample output"
-    The primary version of the application `wisdom-0` will output the following:
-
-    ```
-    app-version: wisdom-0
-    ```
-
-Note that the model version responding to each inference request is noted in the response header `app-version`. In the requests above, we display only this header.
-
 ## Deploy candidate
 
-Deploy a candidate model using a second `InferenceService`:
+A candidate model can be deployed by simply adding a second version to the list of versions comprising the application:
 
 ```shell
-cat <<EOF | kubectl apply -f -
-apiVersion: "serving.kserve.io/v1beta1"
-kind: "InferenceService"
-metadata:
-  name: wisdom-1
-  labels:
-    app.kubernetes.io/name: wisdom
-    app.kubernetes.io/version: v1
-    iter8.tools/watch: "true"
-  annotations:
-    serving.kserve.io/deploymentMode: ModelMesh
-    serving.kserve.io/secretKey: localMinIO
-spec:
-  predictor:
-    model:
-      modelFormat:
-        name: sklearn
-      storageUri: s3://modelmesh-example-models/sklearn/mnist-svm.joblib
+cat <<EOF | helm upgrade --install wisdom $CHARTS/release -f -
+environment: kserve-modelmesh-istio
+application: 
+  metadata:
+    labels:
+      app.kubernetes.io/name: wisdom
+    annotations:
+      serving.kserve.io/secretKey: localMinIO
+  modelFormat: sklearn
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    storageUri: "s3://modelmesh-example-models/sklearn/mnist-svm.joblib"
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v1
+    storageUri: "s3://modelmesh-example-models/sklearn/mnist-svm.joblib"
+  strategy: blue-green
 EOF
 ```
 
 ??? note "About the candidate"
-    In this tutorial, the model source (field `spec.predictor.model.storageUri`) for the candidate is the same as the one for the primary version of the model. In a real world example, this would be different. The version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
-
-## Verify routing changes
-
-The deployment of the candidate model triggers an automatic reconfiguration by Iter8. Inspect the `VirtualService` to see that the routing has been changed. Requests are now distributed between the primary and candidate:
-
-```shell
-kubectl get virtualservice wisdom -o yaml
-```
+    In this tutorial, the model source (field `application.veresions[1].storageUri`) for the candidate is the same as the one for the primary version of the model. In a real world example, this would be different. Here, the version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
```

Comment:
```suggestion
    In this tutorial, the model source (field `application.version[1].storageUri`) for the candidate is the same as the one for the primary version of the model. In a real world example, this would be different. Here, the version label (`app.kubernetes.io/version`) can be used to distinguish between versions.
```

Reply:
I just want to mention that while this has a full path, in these two examples, we only use a partial path.

https://github.com/iter8-tools/docs/pull/119/files#r1364018370
https://github.com/iter8-tools/docs/pull/119/files#r1364019078

I understand that in this section, we are describing a specific version. I just want to make sure you are aware of this.

Reply:
made them all the same